Makefile

```diff
@@ -334,7 +334,7 @@ pre-format: unset-override
 
 format: pre-format
 	@cargo fmt
-	@cargo sort -w ./Cargo.toml ./*/Cargo.toml components/*/Cargo.toml cmd/*/Cargo.toml >/dev/null 
+	@cargo sort -w >/dev/null 
```

Comment:
If some packages are not in workspace members, will they also be sorted?

Reply:
I don't know, I only know some tomls are not sorted with the old command, e.g. "components/cloud/gcp/Cargo.toml".